extensions/sql-migration/src/dashboard/sqlServerDashboard.ts

```diff
@@ -709,24 +712,38 @@ export class DashboardWidget {
 			}
 		});
 
-		const links = [{
-			title: loc.HELP_LINK1_TITLE,
-			description: loc.HELP_LINK1_DESCRIPTION,
-			link: 'https://docs.microsoft.com/azure/azure-sql/migration-guides/managed-instance/sql-server-to-sql-managed-instance-assessment-rules'
-		}];
-
-		linksContainer.addItems(links.map(l => this.createLink(view, l)), {
-			CSSStyles: {
-				'margin-bottom': '8px'
-			}
-		});
-
-		const videosContainer = this.createVideoLinkContainers(view, []);
-		linksContainer.addItem(videosContainer, {
-			CSSStyles: {
-				'margin-bottom': '8px'
-			}
-		});
+		const links = [
+			{
+				title: localize('sql.migration.dashboard.help.link.migrateUsingADS', 'Migrate databases using Azure Data Studio'),
+				description: localize('sql.migration.dashboard.help.description.migrateUsingADS', 'The Azure SQL Migration extension for Azure Data Studio provides capabilities to assess, get right-sized Azure recommendations and migrate SQL Server databases to Azure.'),
+				link: 'https://docs.microsoft.com/en-us/azure/dms/migration-using-azure-data-studio'
```

Comment:
url links should not include the locale.
https://docs.microsoft.com/en-us/azure/dms/migration-using-azure-data-studio
should be
https://docs.microsoft.com/azure/dms/migration-using-azure-data-studio
they will get redirected to the proper page from here

Reply:
same for other urls below

Reply:
@brian-harris good catch, agreed we should remove the locale.

Reply:
thanks Brian, I've removed the locales in the all the links